src/externs/select2.js

```diff
@@ -27,6 +27,15 @@ select2.Event.prototype.params;
  */
 jQuery.prototype.select2 = function(opt_params) {};
 
+/** @type {Object} */
```

Comment:
Though Object is typed, amd is declared to be a function which does not cope with the type annotation. You probably want a namespace.